README.md

```diff
@@ -22,6 +23,10 @@
 
 We auto-deploy `master` branch to [hypha.coop][website] and to the decentralized web with the [Distributed Press][distributed-press]. (See [`deploy.yml`][actions-conf] for how we use GitHub Actions to auto-deploy.)
 
+We use HTMLProofer to check the HTML output but it will also fail on links that are no longer exist, requires login, and on dweb. 
```

Comment:
```suggestion
We use HTMLProofer to check the HTML output, however it will also fail on links that no longer exist, require a login, or are on the dweb. 
```